### DISCUSSION TITLE
Can't get BT Address

### DISCUSSION THREAD

**Discussion by 3xtraFine** [darthcloud/BlueRetro#936] [Q&A] [answered]
Sup guys!
I'm trying to get the BT address to connect to Dualshock 3 but I get the string "getBdAddr [object BluetoothRemoteGATTService]".
I tried changing the firmware versions, now it’s 1.9.2, the result is the same on all firmwares
What am I doing wrong?
![изображение](https://github.com/darthcloud/BlueRetro/assets/83586896/20b78245-dc02-431b-a239-1dd449ad071a)

**Comment by lininjim2** [comment, marked as answer]
![Screensho2 t_2](https://github.com/darthcloud/BlueRetro/assets/107305829/b11538c7-bee8-45df-9c00-d988445bd7e9)
This is what you are looking for

**Reply by 3xtraFine** [reply]
Oh, facepalm... tnx
